packages/saber/vue-app/create-app.js

```diff
@@ -54,15 +54,26 @@ export default context => {
       if (!htmlAttrs.lang) {
         delete htmlAttrs.lang
       }
+      
+      const title = this.$siteConfig.title || 'Saber'
+
+      const defaultMeta = [
+        {
+          name: 'generator',
+          content: `Saber v${__SABER_VERSION__}`
+        },
+        {
+          name: 'description',
+          content: `${this.$siteConfig.description || 'A framework for building modern static websites'}`
+        }
+      ]
```

Comment:
I guess we don't need to set a default title and description.

Reply:
> I guess we don't need to set a default title and description. They're irrelevant to user's project.

Sorry for the delay, I will add conditional statements, check it again. :)